### PR TITLE
Proposal: IPFS+FIL NFT Pinner

### PR DESCRIPTION
**Pull request by momack2**
This is a project proposal for the web3 dev team.

### CODE REVIEW

**Comment by terichadbourne** [comment]
@momack2 May I suggest adding a definition of NFT in this proposal? 🙏

File: proposals/ipfs-fil-nft-pinner.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+Good project scope aims for ~3-5 engineers for 1-3 months (though feel free to suggest larger-scoped projects anyway). 
+Projects do not include regular day-to-day maintenance and improvement work, e.g. on testing, tooling, validation, code clarity, refactors for future capability, etc.
+-->
+<!--
+For ease of discussion in PRs, consider breaking lines after every sentence or long phrase.
+-->
+
+## Purpose &amp; impact 
+#### Background &amp; intent
+There should be a site/API that enables easy pinning for NFT creators to store IPFS-addressed NFTs to Filecoin similar to ipfs2arweave.com
```
Comment (momack2) [inline proposals/ipfs-fil-nft-pinner.md:25]:
```suggestion
An NFT, or non-fungible token, is a cryptocurrency token that is indivisible and unique. NFTs enable creating and trading **digital assets** — such as original artwork, fonts, or blockchain-integrated collectibles like CryptoKitties.

There should be a site/API that enables easy pinning for NFT creators to store IPFS-addressed NFTs to Filecoin similar to ipfs2arweave.com 
```

Comment (terichadbourne) [reply proposals/ipfs-fil-nft-pinner.md:25]: So helpful, thank you!

**Comment by mikeal** [comment]
Closing in favor of https://github.com/protocol/web3-dev-team/pull/62

**Pull request closed without merging**